torch/_inductor/ir.py

```diff
@@ -2259,10 +2259,14 @@ def unflatten_args(new_tensor_args, new_non_tensor_args):
                     new_args.append(next(it_non_tensors))
             return pytree.tree_unflatten(new_args, args_spec)
 
-        tensor_args = [
-            cls.require_contiguous(cls.realize_input(x)) for x in tensor_args
-        ]
-
+        if kernel == torch.ops.aten.convolution:
+            tensor_args = [
```

Comment:
I think stride1 is already required here : https://github.com/pytorch/pytorch/blob/master/torch/_inductor/ir.py#L3011

Reply:
Yes, it is here, https://github.com/pytorch/pytorch/pull/87650 is ok for fixing the channels_last issue, I will check it.